apps/src/code-studio/pd/form_components/QuestionsTable.story.jsx

```diff
@@ -43,14 +43,14 @@ TestWrapper.propTypes = {
 };
 
 export default {
-  title: 'FormComponents/QuestionsTable',
+  title: 'FormComponents/QuestionsTable', // eslint-disable-line storybook/no-title-property-in-meta
```

Comment:
I think this is an example of a component for which the default/named exports doesn't work. You said this story is generated in a loop - do you mind explaining what you mean by this? Just wondering how we can prevent this in the future.

Reply:
This actually isn't an example of that -- this is just allowing the use of a `title` property that someone configured manually. I decided to leave those in place if someone had added a title with a directory structure (ie, has a slash in it).

An example of the loop thing is here:

https://github.com/code-dot-org/code-dot-org/blob/c13389b41c82801f74e0cd890f6e02f381d7e07b/apps/src/templates/progress/ProgressBubble.story.jsx#L17-L37

The problem is that you can only add an export in module scope (ie, the top level in a file, not inside a function call), which isn't possible if you're generating stories in a loop. You can check out the export in that file, which includes the stories as part of the default export (instead of as named exports like we typically do in .story files). I'm actually not 100% sure why that structure works, but it seems to...

Reply:
Gotcha! This makes sense now `/* eslint-disable storybook/default-exports */` - thanks!

Reply:
👍 yep -- and realizing I think it makes sense that this export structure works...we provide the default export (assigned to the key `default` in `module.exports`), and named exports for every property in that `stories` object